erc20/verification.k

```diff
@@ -4,4 +4,19 @@ requires "../lemmas.k"
 module VERIFICATION
     imports EDSL
     imports LEMMAS
+
+ // some treatments for solidity bool variables
+    syntax Bool ::= #rangeBool(Int)  [function]
+                  | #isBoolWord(Int) [function, smtlib(isBoolWord)]
```

Comment:
@tomtomjhj where is the definition of `#isBoolWord`?

Reply:
No rules for `#isBoolWord`. The only purpose of this term is to provide the fact that a symbolic `Int` term is a solidity bool variable, so that the lemmas below can be applied safely.